AN00209_xCORE-200_DSP_Library/app_adaptive/Makefile

```diff
@@ -1,22 +1,32 @@
+BUILD_FLAGS = -O2 -g
+
+XCOREAI ?= 0
+ifeq ($(XCOREAI),0)
```

Comment:
I thought that we agreed to using `ifeq ($(XCOREAI),1)` with the corresponding clause building for the XCORE-AI Explorer and the else clause building for XCORE-200 Explorer.

Reply:
This has been fixed by https://github.com/xmos/lib_dsp/pull/193